### DISCUSSION TITLE
Screen sharing for self-hosted instances

### DISCUSSION THREAD

**Discussion by etadex** [stoatchat/discussions#796] [Feature Suggestions]
The title speaks for itself. I would really like to use this as a working space with my coding team, I have a machine to host it, and traffic is unlimited. It would be really cool to have the option to enable webcams or share a screen during voice chat, just like in discord. 
Pls consider that option atleast for self-hosted solutions?

**Comment by clockwinder** [comment]
This is the only feature I'm waiting for before I switch over to Revolt.

**Comment by TrazhCant** [comment]
Same here.  I understand these kinds of projects aren't fueled by giant budgets and an unlimited pool of skilled brain trust to just pop it all out like a magic wishing lamp.  I always wonder about what donation support would look like on any given open-source project that would have any meaningful impact on development roadmap progress.  (I'm sure it's going to be subjective to most projects).  Of course a deluge of "donation infinitum" is useless if there isn't commensurate increases of dev contributers pledging meaningful commitments as well.  That's always the "big enchilada" I suppose.   Many <3's to all supporters/devs/etc of Revolt as well as everybody involved at any level in the open-source space in general.  Humans kick a55.  :)
